test/galaxy_selenium/navigates_galaxy.py

```diff
@@ -141,6 +141,9 @@ def wait_length(self, wait_type):
 
     def sleep_for(self, wait_type):
         time.sleep(self.wait_length(wait_type))
+    
+    def sleep_for_exactly(self, duration):
+        time.sleep(duration)
```

Comment:
Intentional? Seems unused. I'm fine if you are using it downstream though, but do you want to change the definition of sleep_for to use sleep_for_exactly - removes some duplication and then the entry point is being used in the current code also?

```python
    def sleep_for(self, wait_type):
        self.sleep_for_exactly(self.wait_length(wait_type))
```

I'd also prefer the name ``sleep_for_duration`` or ``sleep_for_seconds`` but the current name is fine if you'd rather not change it.

Reply:
Ahh I use it periodically during busted test diagnosis. I can rename it if you like. sleep_for_seconds sounds fine to me.